backend/config/default.cjs

```diff
@@ -180,4 +180,9 @@ module.exports = {
       registry: 'registry',
     },
   },
+
+  instrumentation: {
+    enabled: false,
+    serviceName: 'backend',
```

Comment:
This should probably include `authenticationToken`.  The config should include every possible field, even if it defaults to empty.